Auth0/SafariWebAuth.swift

```diff
@@ -196,12 +211,21 @@ class SafariWebAuth: WebAuth {
     }
 
     func handler(_ redirectURL: URL) -> OAuth2Grant {
-        if self.responseType.contains([.code]) {
-            var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        var authentication = Auth0Authentication(clientId: self.clientId, url: self.url, telemetry: self.telemetry)
+        if self.responseType.contains([.code]) { // both Hybrid and Code flow
             authentication.logger = self.logger
-            return PKCE(authentication: authentication, redirectURL: redirectURL, reponseType: self.responseType, nonce: self.nonce)
+            return PKCE(authentication: authentication,
+                        redirectURL: redirectURL,
+                        reponseType: self.responseType,
+                        nonce: self.nonce,
+                        leeway: self.leeway,
+                        maxAge: self.maxAge)
         } else {
```

Comment:
**Nit:** `else` not required

Reply:
Will remove it. Thanks for pointing it out.

Reply:
Removed in ca28451be12118c8928745aead45feaa63e669e1.